### PR TITLE
[SDK-2058] Remove v1/event code

### PR DESCRIPTION
**Pull request by nsingh-branch**
## Reference
SDK-2058 -- Remove v1/event code

## Summary
Removed deprecated v1/events `userCompletedAction` and `sendCommerceEvent` code from `BranchSDK.java`, `BranchSDK.h` and `BranchSDK.m`, and `index.js`, as well as from the sample apps. The changes are published in npm under the alpha package `5.2.1-alpha.0`.

## Motivation
Remove old code that has been deprecated for a sufficient amount of time.

## Type Of Change

- [x] Bug fix (non-breaking change which fixes an issue)

## Testing Instructions
Add the alpha package (`5.2.1-alpha.0`) to your app, or run the examples and ensure all functionality is still working.

cc @BranchMetrics/saas-sdk-devs for visibility.

### CODE REVIEW

**Review by echo-branch — COMMENTED** [review]
Correction, comment was meant for another PR.

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "branch-cordova-sdk",
   "description": "Branch Metrics Cordova SDK",
   "main": "src/index.js",
-  "version": "5.2.1",
+  "version": "5.2.1-alpha.0",
```
Comment (gdeluna-branch) [inline package.json:5]: Remember to revert this

**Comment by gdeluna-branch** [comment]
Because we're deleting public methods, this seems like a good time to also update the native ios major.

**Review by gdeluna-branch — CHANGES_REQUESTED** [review]
Revert ver and ticket updating native ios. Please and ty.

**Comment by nsingh-branch** [comment]
> Revert ver and ticket updating native ios. Please and ty.

@gdeluna-branch Just reverted that package.json change and created a ticket to update the native SDK versions.

**Review by gdeluna-branch — APPROVED** [review]

**Pull request merged**